inc/options/class-wpseo-option-wpseo.php

```diff
@@ -345,6 +347,8 @@ protected function validate_option( $dirty, $clean, $old ) {
 					break;
 
 				case 'permalink_structure':
+				case 'category_base_url':
+				case 'tag_base_url':
```

Comment:
Thinking better at this: aren't these values always retrieved from the database thus already sanitized by WordPress?

Reply:
After conversation with Jimmy: there are two cases to consider:
- when users manually change these values, the values are already "sanitized" by WordPress and then we retrieve them from the database: in this case there wouldn't be the need to sanitize them again
- however, when these values are changed directly in the database or by any other change we can't detect and the WP-Cron job runs, we don't know if they're already sanitized: in this case we do need to sanitize them
Basically, placing them in this validation is correct.